api/src/runtime.rs

```diff
@@ -115,25 +118,6 @@ mod tests {
         tests::{new_test_context, TestContext},
     };
 
-    #[test]
-    fn test_bootstrap_failed_if_jsonprc_and_api_configured_same_port_and_different_host() {
```

Comment:
remove this test, because different ip with same port will be served now, and it's a little bit complex to ensure we can get 2 ips for the machine in a unit test, thus we don't test the case anymore.